test/benchmark/ROUTE_PROPAGATION_METRICS.md

```diff
@@ -0,0 +1,224 @@
+# Route Propagation Metrics Enhancement
+
+This document describes the enhancements made to the Envoy Gateway benchmark tests to include route propagation timing metrics, providing visibility into how quickly configuration changes take effect.
+
+## Overview
+
+The benchmark tests now measure **route propagation time** in addition to the existing load test performance and resource utilization metrics. This addresses a critical operational need for understanding configuration velocity and deployment timing.
+
+## New Metrics
+
+The enhanced benchmark reports now include a "Route Propagation Metrics" section with the following timing measurements:
+
+### Timing Definitions
+
+1. **RouteAccepted Duration**: Time from route creation to `RouteConditionAccepted=True`
+   - Measures control plane processing speed
+   - Includes validation, admission, and status updates
+   - Typically 1-5 seconds for small to medium route sets
+
+2. **DataPlaneReady Duration**: Time from `Accepted=True` to traffic flowing correctly
```

Comment:
recommend using existing noun `RouteReady` to replace `DataPlaneReady` and `End-To-End`
 https://gateway-api.sigs.k8s.io/geps/gep-1364/?h=status

RouteReady Duration - T(Apply) -> T(Route in Envoy / 200 Status on Route Traffic)